Makefile

```diff
@@ -1,4 +1,4 @@
-EXTENSIONS := "rv*" "unratified/rv*"
+EXTENSIONS := "/rv*" "unratified/rv*"
```

Comment:
Confused why we need the `/` on the ratified extensions but not before the `unratified` directory?

Reply:
Naively, I would've thought this would become `EXTENSIONS := "extensions/rv*" "extensions/unratified/rv*"`

Reply:
@aswaterman yes, you are correct, there is no need of putting a leading `/` here, and it will work correctly without adding it, @Myrausman you can verify the same.

Reply:
> Naively, I would've thought this would become EXTENSIONS := "extensions/rv*" "extensions/unratified/rv*"

@aswaterman but if we want to use explicit `extensions`, we have to probably use `EXTENSIONS := "../extensions/rv*" "../extensions/unratified/rv*"` using `relative path` as per the what `UNIX` like system says,

by adding `../` we are referring to the `extensions` folder by changing the present working directory `(pwd)` as `root` to `extensions` one, that's why required so.

Reply:
but yes I do agree with @aswaterman by explicit mentioning of `extensions` folder

Reply:
@Myrausman after whatever the changes being accepted, could you please verify also once, that whether the outputs that are generating with these changes are exactly matching with the existing structure, then we are all good to go!

Reply:
> Naively, I would've thought this would become `EXTENSIONS := "extensions/rv*" "extensions/unratified/rv*"`

Yes, I tried it, and it wasn't producing any output in instr_dict.json. Therefore, I have proceeded with changes as follows: `EXTENSIONS := "../extensions/rv*" "../extensions/unratified/rv*"`

but the previous code as below was also working  and producing output:
`EXTENSIONS := "rv*" "unratified/rv*"`